Dockerfile

```
@@ -1,12 +1,12 @@
 ############# builder
-FROM eu.gcr.io/gardener-project/3rd/golang:1.16.5 AS builder
+FROM eu.gcr.io/gardener-project/3rd/golang:1.17.5 AS builder
 
 WORKDIR /go/src/github.com/gardener/gardener-extension-shoot-dns-service
 COPY . .
 RUN make install
 
 ############# gardener-extension-shoot-dns-service
-FROM eu.gcr.io/gardener-project/3rd/alpine:3.13.5 AS gardener-extension-shoot-dns-service
+FROM 3.15.0 AS gardener-extension-shoot-dns-service
```

```suggestion
FROM alpine:3.15.0 AS gardener-extension-shoot-dns-service
```
 Too aggressive shortening ;-)